ReactWindows/ReactNative/Views/Image/ReactImageManager.cs

```diff
@@ -273,21 +320,36 @@ private void OnImageStatusUpdate(Border view, ImageLoadStatus status, ImageMetad
                 var imagePipeline = ImagePipelineFactory.Instance.GetImagePipeline();
                 var image = default(BitmapSource);
                 var uri = new Uri(source);
+                var builder = ImageRequestBuilder.NewBuilderWithSource(uri);
+                var imageData = GetImageData(view);
 
-                // Remote images
-                if (source.StartsWith("http:") || source.StartsWith("https:"))
+                // Apply postprocessor if Composition isn't supported.
+                if (imageData.BackgroundColor != null || 
+                    (imageData.TintColor != null && !EffectManager.IsCompositionSupported))
```

Comment:
It looks like the TintColor will always be applied by the postprocessor if a BackgroundColor is set, even if the Composition is enabled.